package.json

```diff
@@ -25,7 +25,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint .  --ext .js"
+    "lint": "eslint '**/*.{js,jsx}'",
```

Comment:
normally if you're using eslint and prettier you will be combining the integrations (google prettier eslint)

for short term, delete lint, try adding it in another PR